index.html

```diff
@@ -12,10 +12,57 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+    <header>
+        <div class="logo">
+          
+          <img class="logo" src="http://swarley00.github.io/startup-landing-page-clone/images/karma-logo.svg" alt="Logo Image">
+            
+        </div>
+        <nav class = "nav-bar">
+            <ul>
+                <li>Meet Karma</li>  <li>How-it-Works</li> 
+ <li>Store</li> 
```

Comment:
Be careful, indentation is really important to be able to read code and spot mistakes easily